src/styles/_variables.scss

```
@@ -0,0 +1,3 @@
+$cardWidth: 200px;
+$cardHeight: 300px;
+$cardGap: 48px;
```

let's stick to this dash case
```suggestion
$card-width: 200px;
$card-height: 300px;
$card-gap: 48px;
```